src/lib/deploy.js

```diff
@@ -449,19 +450,40 @@ async function deploy({ alias, yes }) {
     return;
   }
 
-  const txUrl = `https://berkeley.minaexplorer.com/transaction/${txn.data.sendZkapp.zkapp.hash}`; // TODO: Make the network configurable
   const str =
     `\nSuccess! Deploy transaction sent.` +
     `\n` +
     `\nNext step:` +
     `\n  Your smart contract will be live (or updated)` +
     `\n  as soon as the transaction is included in a block:` +
-    `\n  ${txUrl}`;
+    `\n  ${getTxUrl(graphQLEndpoint, txn)}`;
 
   log(green(str));
   await shutdown();
 }
 
+// Get the desired blockchain explorer url with txn hash
+function getTxUrl(graphQLEndpoint, txn) {
+  const explorerName = graphQLEndpoint
+    .split('.')
+    .filter((item) => (item === 'minascan') | (item === 'minaexplorer'))?.[0];
+  let txBaseUrl;
+
+  switch (explorerName) {
+    case 'minascan':
+      txBaseUrl = `https://minascan.io/berkeley/zk-transaction/`;
+      break;
+    case 'minaexplorer':
+      txBaseUrl = `https://berkeley.minaexplorer.com/transaction/`;
+      break;
+    default:
+      txBaseUrl = `https://berkeley.minaexplorer.com/transaction/`;
```

Comment:
Thinking if we should make the fallback case random among the two options. Although it won't currently be encountered ever

Reply:
That's a great idea!